packages/stepper/src/EllipsesStep/styles.ts

```diff
@@ -0,0 +1,35 @@
+import { css } from '@leafygreen-ui/emotion';
+import { palette } from '@leafygreen-ui/palette';
+import { spacing } from '@leafygreen-ui/tokens';
+import { stepIconClassName } from '../constants';
+import { StepStates } from '../types';
+import { EllipsesStepStates } from './types';
+
+// TODO: would be good to put these styles inside a baseStyles, but it is currently impossible because the <Tooltip> content is an iframe.
```

Comment:
I think I commented on this in the original PR, but Tooltip is in a Portal, not an iframe. Not sure how that affects our ability to style it